OneBlink.SDK/models/FormSubmissionApprovalNote.cs

```diff
@@ -0,0 +1,30 @@
+namespace OneBlink.SDK.Model
+{
+    public class FormSubmissionApprovalNote
+    {
+        public string id
+        {
+            get; set;
+        }
+        public string createdAt
```

Comment:
Should these be ` DateTime`s?

Reply:
Both were being used, changed to DateTime's